src/angular-business/sidebar/sidebar.md

```diff
@@ -0,0 +1,354 @@
+<sbb-sidebar-container>
+  <sbb-sidebar-content>Main</sbb-sidebar-content>
+  <sbb-sidebar-content>Main 2</sbb-sidebar-content>
+</sbb-sidebar-container>
+```
+
+```html
+<!-- Invalid because the `<sbb-sidebar>` is outside of the `<sbb-sidebar-container>`. -->
+<sbb-sidebar-container></sbb-sidebar-container>
+<sbb-sidebar></sbb-sidebar>
+```
+
+### Sidebar Navigation Content
+
+Inside a `<sbb-sidebar>` it's possible to place `<sbb-expansion-panel>` and `<fieldset>` elements in which you can place every content you like.
+Any different direct descendant of `<sbb-sidebar>` than the two mentioned elements are being ignored.
+
+Primarly it is intended to place links with the attribute selector `sbbSidebarLink` inside of `<sbb-expansion-panel>` and `<fieldset>` to achieve a navigation.
+
+To display the active state correctly, use the css class `sbb-sidebar-link-active`.
+For example if using with a routerLink, write `routerLinkActive="sbb-sidebar-link-active"`.
+
+#### Example with angular router
+
+```html
+<sbb-sidebar-container>
+  <sbb-sidebar role="navigation">
+    <sbb-expansion-panel expanded>
+      <sbb-expansion-panel-header>Introduction</sbb-expansion-panel-header>
+      <a sbbSidebarLink routerLink="/getting-started" routerLinkActive="sbb-sidebar-link-active"
+        >Getting Started</a
+      >
+      <a sbbSidebarLink routerLink="/typography" routerLinkActive="sbb-sidebar-link-active"
+        >Typography</a
+      >
+    </sbb-expansion-panel>
+    <fieldset>
+      <legend>Fieldset Example</legend>
+      <button sbbButton mode="primary">Random Content</button>
+    </fieldset>
+  </sbb-sidebar>
+  <sbb-sidebar-content role="main">
+    <router-outlet></router-outlet>
+  </sbb-sidebar-content>
+</sbb-sidebar-container>
+```
+
+### Opening and closing a sidebar
+
+A `<sbb-sidebar>` can be opened or closed using the `open()`, `close()` and `toggle()` methods. Each
+of these methods returns a `Promise<boolean>` that will be resolved with `true` when the sidebar
+finishes opening or `false` when it finishes closing.
+
+The opened state can also be set via a property binding in the template using the `opened` property.
+The property supports 2-way binding.
+
+`<sbb-sidebar>` also supports output properties for just open and just close events, The `(opened)`
+and `(closed)` properties respectively.
+
+### Setting the sidebar's size
+
+The `<sbb-sidebar>` will, by default, have an expanded width of 300px. The width can
+be explicitly set via CSS:
+
+```css
+.sbb-sidebar {
+  width: 250px;
+}
+```
+
+If you like fit the size of its content, just set css `width` to `auto`.
+
+Try to avoid percent based width as `resize` events are not (yet) supported.
+
+### Using with sbb header
+
+If you like to use the sidebar after the `<sbb-header>`, please
+apply the css class `sbb-sidebar-after-header` to the `<sbb-sidebar-container>`.
+
+```html
+<sbb-header>...</sbb-header>
+<sbb-sidebar-container class="sbb-sidebar-after-header">
```

Comment:
Should we add a convenience when the siderbar or icon-sidebar is placed immediately after the sbb-header?